app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@GregP As this discussion got sidetracked to other questions, be sure to remember that the original question here is whether or not a virtual column is needed, to which the answer seems to be "no".